interfaces/builtin/content.go

```diff
@@ -78,6 +78,16 @@ func (iface *contentInterface) SanitizeSlot(slot *interfaces.Slot) error {
 		slot.Attrs["content"] = slot.Name
 	}
 
+	// Error if "read" or "write" are present alongside "source".
+	if _, ok := slot.Attrs["source"]; ok {
+		if _, ok := slot.Attrs["read"]; ok {
+			return fmt.Errorf(`move the "read" attribute into the "source" section`)
```

Comment:
Maybe a bit more context here? i.e. `move the "read" attribute of slot %q into the "source" section` and pass slot.Name in ? Same below.